src/commands/createWebApp/WebAppCreateStep.ts

```diff
@@ -49,6 +51,8 @@ export class WebAppCreateStep extends AzureWizardExecuteStep<IWebAppWizardContex
             siteConfig: newSiteConfig,
             reserved: context.newSiteOS === WebsiteOS.linux  // The secret property - must be set to true to make it a Linux plan. Confirmed by the team who owns this API.
         });
+        // Sets "showBuildDuringDeployPrompt" to false --> SCM_DO_BUILD_DURING_DEPLOYMENT(set to true) is added as a appSetting by default for new web apps
+        await updateWorkspaceSetting(constants.configurationSettings.showBuildDuringDeployPrompt, false, undefined);
```

Comment:
This doesn't really fix the root issue. Ideally our logic that prompts about `SCM_DO_BUILD_DURING_DEPLOYMENT` should check the app settings at that time. This doesn't handle people that create an app or change the app setting outside VS Code.